code/game/objects/items/weapons/tools.dm

```diff
@@ -273,6 +273,24 @@
 		else
 			user << "<span class='notice'>You are already trying to pull out a teeth!</span>"
 		return
+	if(ishuman(C) && C.dna.species.id == "wire" && C != user)

```

Comment:
Snowflake roundstart races should be humans mechanically. Introducing bad game design like this is detrimental.